src/core/inlines.js

```diff
@@ -77,10 +77,13 @@ const inlineElement = /(?:\[\^[^^]+\^\])/; // Inline [^element^]
  */
 function inlineElementMatches(matched) {
   const value = matched.slice(2, -2).trim();
-  const [element, attribute] = value.split("/", 2).map(s => s && s.trim());
+  const [element, attribute] = value
+    .replace("\\/", "%%")
+    .split("/", 2)
+    .map(s => s && s.trim().replace("%%", "/"));
   const [xrefType, xrefFor, textContent] = attribute
     ? ["element-attr", element, attribute]
-    : ["element", null, element];
+    : ["element|element-attr", null, element];
```

Comment:
Does this mean there is no element-only syntax anymore?

Reply:
Yes. There isn't. 
Maybe we can add `element-attr` only if `value.includes("\\/")`, because elements don't have slashes. Ideally, this should be `attr-value`, but Bikeshed/specs have data messed up.

Reply:
Could you give me an example for me to understand better? 👀

Reply:
The example in the referenced issue is one: `application/x-www-form-urlencoded`. It's not an attribute (`element-attr`), but an attribute value `attr-value` for `enctype` attribute.

> Maybe we can add element-attr only if value.includes("\\/"), because elements don't have slashes.

This is partially wrong assumption, as it can be `element-attr` without `/`

Reply:
How about we support this:
```
[^element^], [^element/element-attr^] // general, good usage
[^/element-attr^] // for terms like above
(we don't support attr-value in ReSpec yet, might in near future)
```

Reply:
> but HTML exported it as (or Bikeshed understood it to be) `element-attr`.

Shouldn't we fix HTML instead of doing `element|element-attr`? This type of mess shouldn't be frequent.

Reply:
Good point. I wonder why I jumped on the fix :flushed:.
I just checked, and there are only 3 instances near https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#attr-fs-enctype:

```
    The "application/x-www-form-urlencoded" keyword and corresponding state.
    The "multipart/form-data" keyword and corresponding state.
    The "text/plain" keyword and corresponding state.
```

Reply:
> Good point. I wonder why I jumped on the fix 😳.

This is why a peer review is essential even for obvious changes 😁

Reply:
Filed https://github.com/whatwg/html/issues/5586